ansible_navigator/config.py

```diff
@@ -74,11 +94,11 @@ def get(self, keys: List[str], default: Any = Sentinel) -> Any:
             else:
                 break
         else:
-            return current
+            return "user provided config file", current
```

Comment:
Instead of string maybe we can use enums? For eg:
```
class ConfigSource():
    NONE = 0
    # user provided config file
    USER_CONFIG_FILE = 1
    # argparse default config
    ARGPARSE_DEFAULT = 2
    # default config
    DEFAULT_CONFIG = 3
```